sdk/wallet-adapter/adapters/base-adapter/src/index.ts

```diff
@@ -27,16 +27,22 @@ export interface WalletAdapter {
     event: E,
     callback: WalletAdapterEvents[E]
   ) => () => void;
-  signTransaction(transaction: SignableTransaction): Promise<SignedTransaction>;
+  signTransaction(transactionInput: {
+    transaction: SignableTransaction;
+    account: string;
+    chain: string;
```

Comment:
Can we make this optional here while we still figure out our chain story? I think it should be required in the standard, but for dapps we'll just pull a default value out of the `WalletAccount` for now.